File: 1378-cells-with-odd-values-in-a-matrix/1378-cells-with-odd-values-in-a-matrix.java
class Solution {
    public int oddCells(int m, int n, int[][] indices) {
        int []row = new int[m];
        int []col = new int[n];

        for (int []idx : indices){
            row[idx[0]]++;
            col[idx[1]]++;
        }
        int count = 0;
        for (int i = 0;i < m;i++){
            for (int j = 0;j < n;j++){
                if((row[i] + col[j]) % 2 == 1) {
                    count ++;
                }
            }
        }
    return count;

    }
}

File: 1378-cells-with-odd-values-in-a-matrix/1378-cells-with-odd-values-in-a-matrix.py
class Solution(object):
    def oddCells(self, m, n, indices):
        row = [0] * m
        col = [0] * n
        for i in indices:
            row[i[0]] += 1
            col[i[1]] += 1
        
        count = 0
        
        for i in range(m):
            for j in range(n):
                if (row[i] + col[j]) % 2 == 1:
                    count += 1
        return count 